web-app/HeimScripts/heatmap/run.R

```diff
@@ -53,3 +54,8 @@ buildFields <- function(df){
   return(df)
 }
 
+fixString <- function(str) {
+  str <- gsub("(?!-)[[:punct:]]", "", str, perl=TRUE)
```

Comment:
In theory it could. 
In practice the only illegal characters I've found so far were in affymetrix probe_ids, probably to denote merged probes hence slash between two IDs. 
Better solution would be to just rewrite D3 part to avoid using Row.labels as CSS selectors - then they could be left at peace as they are.